frontend/src/components/BackButton.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react'
+import { useLocation } from 'react-router-dom'
+import Button from '@mui/material/Button'
+
+const BackButton = () => {
+  const location = useLocation()
+
+  const handleClick = () => {
+    window.history.back()
+  }
+
+  return location.pathname === '/' ? null : (
```

Comment:
Can we also exclude "/explore" and "/ai/creator" paths here?

Reply:
works now